flowise-embed/src/components/buttons/SendButton.tsx

```diff
@@ -41,12 +45,10 @@ export const DeleteButton = (props: SendButtonProps) => {
         `py-2 ${paddingClass} justify-center font-semibold text-white focus:outline-none flex items-center disabled:opacity-50 disabled:cursor-not-allowed disabled:brightness-100 transition-all filter hover:brightness-90 active:brightness-75 chatbot-button ` +
         props.class
       }
-      style={{ background: 'transparent', border: 'none' }}
-      title="Reset Chat"
+      style={{ background: 'transparent', border: '1px solid white', color: 'white', padding: '0.3125rem 1.875rem' }}
+      title="New Chat"
     >
-      <Show when={!props.isLoading} fallback={<Spinner class="text-white" />}>
-        <DeleteIcon color={props.sendButtonColor} class={'send-icon flex ' + (props.disableIcon ? 'hidden' : '')} />
-      </Show>
+      <span style={{ color: 'white' }}>{props.newItemText || '+ NOVO CHAT'}</span>
```

Comment:
Ao invés de deixar o texto default, é preferível manter a prop `newItemText` como obrigatória, para evitar textos dentro de componentes.
Além disso, o "+ " pode fazer parte do próprio conteúdo e deixar apenas o texto para ser passado e também aplicar a propriedade CSS `uppercase` para que a formatação do texto não dependa da string, ficando mais ou menos assim:
```suggestion
      <span style={{ color: 'white'; textTransform: 'uppercase' }}>+ {props.newItemText}</span>